src/main/java/org/broadinstitute/hellbender/utils/read/GATKReadToBDGAlignmentRecordConverter.java

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.hellbender.utils.read;
+
+import htsjdk.samtools.SAMFileHeader;
+import org.apache.spark.api.java.function.Function;
+import org.bdgenomics.formats.avro.AlignmentRecord;
+import org.bdgenomics.adam.converters.SAMRecordConverter;
+import org.bdgenomics.adam.models.SequenceDictionary;
+import org.bdgenomics.adam.models.RecordGroupDictionary;
+
+
+import java.io.Serializable;
+
+/**
+ * Converts a GATKRead to a BDG AlignmentRecord
+ */
+public class GATKReadToBDGAlignmentRecordConverter implements Function<GATKRead, AlignmentRecord> {
```

Comment:
With respect to the static methods, isn't this a Spark issue rather than ADAM?  I have to pass a subclass of `Function` to the `.map` method.

I'm not sure I understand the advantage of a singleton here.  The class is only being instantiated once when the Spark DAG is being constructed.